package.json

```diff
@@ -19,21 +19,24 @@
     "lodash": "^3.5.0"
   },
   "devDependencies": {
+    "async": "^1.5.0",
     "bower": "^1.3.12",
     "coffee-coverage": "^0.7.0",
     "coffee-loader": "^0.7.2",
     "coffee-script": "^1.9.1",
+    "coverage-capture": "./test/coverage-capture",
     "gulp": "^3.8.11",
     "gulp-mocha-phantomjs": "^0.10.1",
+    "gulp-rename": "^1.2.2",
+    "gulp-template": "^3.1.0",
     "gulp-webpack": "^1.3.1",
-    "mocha-phantomjs": "^4.0.1",
-    "should": "^7.1.1",
-    "simulate": "^0.0.5",
-    "webpack": "^1.7.3",
```

Comment:
Think this means that `make` will no longer work. Perhaps it'd make sense to remove the `Makefile`?